### PR TITLE
fix: pass config prop to the renderer

### LINKED ISSUE #49: font size config not working for MathText
config props works for MathView but not working for MathText

```
    <MathText
      value={props.math}
      config={{ ex: 50, em: 50 }}
      onError={e => console.log('ERROR!', e)}
      direction="ltr"
    />
```
This doesn't show big font.
Screenshot at Jul 23 00-29-31

And where can I see all the props?

### PR DESCRIPTION
**Pull request by KhubaibQaiser**
closes #49

### CODE REVIEW

**Review by ShaMan123 — CHANGES_REQUESTED** [review]
Please make changes without formatting diffs

**Comment by KhubaibQaiser** [comment]
> Please make changes without formatting diffs

Yeah, sorry about that. I have fixed the formatting issue.

**Comment by KhubaibQaiser** [comment]
Hi @ShaMan123 , could you please review the changes, merge the PR and push a release. This would help so many waiting for the fix.

File: package.json
Comment (ShaMan123) [inline package.json:1]: please undo this change, npm scripts handle semver changes

File: src/MathText.tsx [hunk 1]
```diff
@@ -42,7 +42,8 @@ export type MathTextProps = Pick<MathTextRowProps, 'direction' | 'containerStyle
 
 export const InlineMathItem = React.memo(({ value, isMath, CellRendererComponent, Component, inline, style, ...props }: MathTextItemProps) => {
     if (value === '') return null;
-    const config = useMemo(() => ({ inline }), [inline]);
+    const { config: configProps } = props as MathTextItemProps<true>;
+    const config = useMemo(() => ({ ...(configProps || {}), inline }), [inline, configProps]);
```
Comment (ShaMan123) [inline src/MathText.tsx:46]: If config is always passed as optional then you should change `MathTextItemProps` it seems.
`config?: MathViewProps['config']`



https://github.com/ShaMan123/react-native-math-view/blob/a089f98a28d278a6a52e8ed6891e114fff50e588/src/common.tsx#L22C22-L22C37

Comment (ShaMan123) [reply src/MathText.tsx:46]: Is that correct?

**Review by ShaMan123 — CHANGES_REQUESTED** [review]
I will add a github publish action

File: src/MathText.tsx (hunk 1 above)
Comment (KhubaibQaiser) [reply src/MathText.tsx:46]: The `MathTextItemProps` extends from `MathViewCommonProps` which defines `config` as optional, which I guess is intentional? Because if you look at the `MathTextItemProps` it optionally extends from `MathViewCommonProps` if the generic type is a boolean. Not sure what the purpose is for this optional extension.

```export type MathTextItemProps<T extends boolean = boolean> = (T extends true ? Omit<MathViewProps, 'math'> : TextProps) & {
    value: string,
    isMath: T,
    Component?: MathView,
    CellRendererComponent?: ElementOrRenderer,
    inline?: boolean
}```

Comment (ShaMan123) [reply src/MathText.tsx:46]: So I prefer to spread `config: configProp` in line 43 if that is possible.
Sorry for the delay and for not being acquainted with the code anymore. 

Actually, I think it is best you fork and I archive this repo with a pinned issue stating so so other devs can write there what they want.

**Comment by ShaMan123** [comment]
#97 

I suggest you fork
